.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
I think this is wrong. Sudo wasn't required before and won't be the default with Trusty. I don't think we need to change this setting.